lib/predictions/pub_sub.ex

```diff
@@ -54,9 +56,9 @@ defmodule Predictions.PredictionsPubSub do
   # Server
 
   @impl GenServer
-  def init(opts) do
-    subscribe_fn = Keyword.get(opts, :subscribe_fn, &Phoenix.PubSub.subscribe/2)
-    subscribe_fn.(Predictions.PubSub, "predictions")
+  def init(_) do
+    Phoenix.PubSub.subscribe(@predictions_phoenix_pub_sub, "predictions")
```

Comment:
question: why are we making this compile-time instead of a runtime option? It's conventional for callbacks like `init` to actually configure the process they're starting dynamically from the options being passed in.

Reply:
Because we're using Mox and determining the implementation at compile time.